src/apps/integrations/loris/service/loris_client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import aiohttp
+from aiohttp.client_exceptions import ClientConnectorError
+
+from apps.integrations.loris.errors import LorisBadCredentialsError, LorisInvalidHostname, LorisInvalidTokenError
+from apps.shared.domain.custom_validations import InvalidUrlError, validate_url
+
+
+class LorisClient:
+    @classmethod
+    async def login_to_loris(self, hostname: str, username: str, password: str) -> str:
+        try:
+            hostname = validate_url(hostname)
+        except InvalidUrlError as iue:
+            raise LorisInvalidHostname(hostname=hostname) from iue
+        timeout = aiohttp.ClientTimeout(total=60)
+        loris_login_data = {
+            "username": username,
+            "password": password,
+        }
+        async with aiohttp.ClientSession(timeout=timeout) as session:
+            try:
+                async with session.post(
+                    f"{hostname}/api/v0.0.3/login",
```

Comment:
Adding note here for completeness:
`hostname` is going to be updated to expect a full url up to the API version, to avoid having to hardcode `/api/v0.0.3` here and when retrieving projects